### ISSUE TITLE
Addressables Reimport API

### ISSUE THREAD

**Issue by Mefodei** [favoyang/unity-addressable-importer#21]
Hello there. 
I try to call package API "Reimport Addressable Group (Check Folders)" on Project Build Execution step. But all general static functions is private. Could you allow use Common method of your package as public API?

**Comment by favoyang** [comment]
Hi @Mefodei,

I assume you mean to make "Assets/AddressablesImporter: Check Folder(s)" publicly available? That method only works for the current selection. Is that suitable for your case?

**Comment by Mefodei** [comment]
Hi
I'm looking for API method, that takes project folder as parameters and apply reimport action. For example it can be all rules folders from asset Addressables Import Settings Rules.

**Comment by favoyang** [comment]
That makes sense to me.

I have prepared a branch with a reimport API, which requires an array of folder asset paths. An asset path is a string starts with "Assets". E.g. to process the whole Assets folder (this can be slow depends on the number of assets).
```
AddressableImporter.FolderImporter.ReimportFolders(new string[] { "Assets" });
```
To test it, switch your lib to this Git URL.

```
{
    "dependencies": {
        "com.littlebigfun.addressable-importer": "https://github.com/favoyang/unity-addressable-importer.git#reimport-api"
    }
}
```

Let me know if this works for you, then I will merge back to the master branch.

**Comment by Mefodei** [comment]
Thx, i will try

**Comment by Mefodei** [comment]
Works fine :)

**Comment by favoyang** [comment]
@Mefodei, merged to master and released `v0.7.0`. The reimport-api branch is deleted.
